test/jdk/jdk/incubator/vector/templates/Kernel-Slice-bop-const.template

```diff
@@ -0,0 +1,11 @@
+        $type$[] a = fa.apply(SPECIES.length());
```

Comment:
Forgot to commit the updated unit tests?

Reply:
This is for the microbenchmarks generated in the panama-vector repo only.
Thanks a lot.